sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -259,6 +304,111 @@ class BlobRestClient
     };
   };
 
+  class PageBlob
+  {
+  public:
+    struct CreateOptions : public Common::RequestOptions
+    {
+      std::string ContentEncoding;
+      std::string ContentLanguage;
+      std::string CacheControl;
+      std::string ContentDiposition;
+      std::map<std::string, std::string> Metadata;
+      AccessTier Tier;
+    };
+
+    static azure::core::http::Request CreateConstructRequest(
+        const std::string& url,
+        const CreateOptions& options)
+    {
+      auto request = azure::core::http::Request(azure::core::http::HttpMethod::Put, url);
+      ApplyBasicHeaders(options, request);
+
+      request.addHeader(k_HEADER_X_MS_BLOB_TYPE, k_HEADER_PAGE_BLOB);
+
+      Common::AddMetadata(options.Metadata, request);
+
+      if (options.Tier != AccessTier::Unknown)
+      {
+        // TODO: wait until enum to string
+        // request.addHeader(Common::k_HEADER_X_MS_ACCESS_TIER, );
+      }
+
+      return request;
+    };
+
+    // TODO: should return azure::core::http::Response<BlobContainerInfo> instead
+    static BlobContentInfo CreateParseResponse(/*const*/ azure::core::http::Response& response)
+    {
+      BlobContentInfo info;
+      if (response.getStatusCode() >= 300U)
+      {
+        /* TODO: when response errored */
+      }
+      else
+      {
+        ParseBasicResponseHeaders(response.getHeaders(), info);
+        info.Etag = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_ETAG);
+        info.LastModified
+            = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_LAST_MODIFIED);
+        info.ContentMD5
+            = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_CONTENT_MD5);
+        info.ContentCRC64
+            = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_X_MS_CONTENT_CRC64);
+        info.SequenceNumber = std::stoull(Common::GetHeaderValue(
+            response.getHeaders(), k_HEADER_X_MS_BLOB_SEQUENCE_NUMBER));
+        info.ServerEncrypted
+            = Common::GetHeaderValue(response.getHeaders(), Common::k_HEADER_X_MS_SERVER_ENCRYPTED)
+            == "true";
```

Comment:
Should `true` also be defined as `k_HEADER_VALUE_TRUE` or `k_HEADER_TRUE`?

Reply:
okay